index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="header">
+        <h2 class="h2">Welcome</h2>
+        <nav>
+            <img class="nav" src="./images/Union.png" alt="">
+        </nav>
+    </header>
+    <section class="intro">
+        <h1 class="h1">
+        Hey there. I’m <br>Richard
+        </h1>
+        <h2 class="sd">
+            I’m a software developer
+        </h2>
+            <p class="prg">
+                I can help you build a product ,
+                 feature or website Look through some of my work and experience!
+                  If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.
+            </p>
+            <ul class="icon-list">
+                <li><img src="./images/twitter.png" alt="twitter"></li>
+                <li><img src="./images/Linkedin.png" alt="linkedin"></li>
+                <li><img src="./images/medium.png" alt="medium"></li>
+                <li><img src="./images/github.png" alt="github"></li>
+                <li><img src="./images/angellist.png" alt="angellist"></li>
```

Comment:
- [ ] Kindly put your images in an `a` tag so as to navigate to your social media when clicked. it should look like this ` <li><a href=''><img src="./images/twitter.png" alt="twitter"></a></li>`